### DISCUSSION TITLE
Suggestion for an alternative way to get Chrome version on Windows

### DISCUSSION THREAD

**Discussion by plu5** [FlareSolverr/FlareSolverr#684] [Ideas]
See [`get_chrome_major_version` in utils.py](https://github.com/FlareSolverr/FlareSolverr/blob/2d97f882764caf6563647109871b5230d5899840/src/utils.py#L95). On Windows it tries to get the version from the registry, but on my system that key did not exist:

    2023-01-28 12:10:30 INFO     Chrome / Chromium path: C:\Users\P\AppData\Local\Google\Chrome\Application\chrome.exe
    ERROR: The system was unable to find the specified registry key or value.
    2023-01-28 12:10:30 ERROR    Chrome / Chromium version not detected!

I suggest an alternative way to get the version on Windows:

```python
chrome_path = uc.find_chrome_executable()
process = os.popen(f'powershell -command "(Get-Command "{chrome_path}").Version.ToString()"')
complete_version = process.read()
process.close()
```

**Comment by ngosang** [comment]
Which version of Windows do you have?

**Reply by plu5** [reply]
10 21H2

Chrome 109.0.5414.120

**Comment by ngosang** [comment]
I fixed it in FlareSolverr 3.1.0
8d9bac9dd435f2bd0f818f0d66b158a35d055c99
